bundles/org.openhab.binding.shelly/doc/UseCaseSmartRoller.md

```diff
@@ -197,23 +209,27 @@ There are two valuable options, which even extend and improve the user experienc
 - Button-1
 
 Using the described setup you could add a Button-1 to trigger rules, e.g.
+
 - 1xshort: open roller
 - 2xshort: go to 25% (because disturbs enjoying your morning coffee with your wife or husband)
 - 3xshort: go to 75% (sun is disturbing while watching your TV)
 - long   : close shutter (if not done by some automation)
 
 - Voice control
+
 And of course you could utilize openHAB's voice integration, e.g. with Alexa, Siri or Google Home.
 
 - Scenes
-There are more of less limits when integrating even more advanced roller control based on openHAB rules. 
+
+There are more of less limits when integrating even more advanced roller control based on openHAB rules.
 This allows to provide a scene to the user, which does a lot of complex stuff under the hood.
 
 Be welcome to contribute your ideas to the community, this is what creates the difference between openHAB/Shelly setup compared to rollers with button control.
 
 ### Resources
 
 There is a bunch of ressources you could browse around openHAB, Shelly or the binding
+
```

Comment:
Can you fix the `ressources` typo in the line above?

Reply:
Yes fixed.